clang/lib/CIR/CodeGen/CIRGenFunction.h

```diff
@@ -582,6 +582,22 @@ class CIRGenFunction : public CIRGenTypeCache {
 
   void buildAggExpr(const clang::Expr *E, AggValueSlot Slot);
 
+  /// buildComplexExpr - Emit the computation of the specified expression of
```

Comment:
We should always remove repetition of method names in the comments, like "buildComplexExpr - " above.